src/routes/members/[studentId]/UpdateMemberForm.svelte

```diff
@@ -46,12 +46,17 @@
       error={$errors.lastName}
     />
   </div>
-  <div class="flex gap-2 [&>*:nth-child(3)]:flex-1">
-    <Labeled label="Program" id="classProgramme" error={$errors.classProgramme}>
+  <div class="flex w-full flex-wrap gap-2 [&>*:nth-child(3)]:flex-1">
+    <Labeled
+      label="Program"
+      id="classProgramme"
+      error={$errors.classProgramme}
+      fullWidth={true}
```

Comment:
This can be simplified:
```suggestion
      fullWidth
```